docs/integrations/gitlab/org.md

```diff
@@ -239,7 +241,8 @@ catalog:
         host: gitlab.com ## Could also be self hosted.
         orgEnabled: true
         group: org/teams # Required for gitlab.com when `orgEnabled: true`. Optional for self managed. Must not end with slash. Accepts only groups under the provided path (which will be stripped)
-        restrictUsersToGroup: true # Backstage will ingest only users directly assigned to org/teams.
+        restrictUsersToGroup: true # Optional: Backstage will ingest only users directly assigned to org/teams.
+        includeUsersWithoutSeat: true # Optional: Include users without paid seat, only valid for SaaS
```

Comment:
```suggestion
        includeUsersWithoutSeat: false # Optional: Include users without paid seat, only valid for SaaS
```
Perhaps we should show `false` in this example as it's likely what most people want but copy paste of config is a good way of getting started for many :)